java/clients/src/main/java/sleeper/clients/admin/TableSelectHelper.java

```diff
@@ -58,7 +58,7 @@ public Optional<TableProperties> chooseTableOrReturnToMain(String instanceId) th
         String tableName = chosen.getEntered();
         try {
             return Optional.of(store.loadTableProperties(instanceId, tableName));
```

Comment:
We don't need to load the table properties for the reports. We can refactor this.